src/shared/sources/_lib/source-map.js

```diff
@@ -3,8 +3,9 @@ const globJoin = require('../../utils/glob-join.js')
 const sourceKey = require('./source-key.js')
 const { sep } = require('path')
 
-module.exports = function sourceMap () {
-  const scrapers = globJoin(__dirname, '..', '**', '*.js')
+module.exports = function sourceMap (params = {}) {
+  const scrapersPath = params._sourcesPath || globJoin(__dirname, '..')
+  const scrapers = globJoin(scrapersPath, '**', '*.js')
   let filePaths = glob(scrapers)
```

Comment:
idk why I wrote this originally as scrapers but shouldn't this probably be named sources? (Law of triviality coming into effect here...)

Reply:
grrrr yes it should, thanks!

Reply:
Fixed.